packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -23,6 +23,7 @@ const logger = new Logger(module)
 
 const SPONSOR_AMOUNT = 200
 const STAKE_AMOUNT = 100
+const cases = [parseEther("0.001"), parseEther("0.0005")]
```

Comment:
This could be inlined to line 92. (Or if we need to use constant, could rename to CASES as it is a constant.)

Reply:
Inlined it.